Modules/install-module.R.in

```diff
@@ -30,6 +30,12 @@ options(
   configure.vars = c(jaspBase = "INCLUDE_DIR='@PROJECT_SOURCE_DIR@/Common'") #Needed for flatpak build as it keeps recompiling jaspBase (which it shouldnt of course but I dont know why) and this is an easy fix to get it to work now
 )
 
+if (jaspBase::getOS() == "osx") {
+	options(pkgType = "mac.binary")
+} else if (jaspBase::getOS() == "windows") {
+	options(pkgType = "win.binary")
+}
```

Comment:
```suggestion
if (jaspBase::getOS() == "osx") {
  options(pkgType = "mac.binary")
} else if (jaspBase::getOS() == "windows") {
  options(pkgType = "win.binary")
}
```

Reply:
maybe we should just make a .vscode-workspace file that sets all these things correctly... not sure if something like that exists for QtCreator though (it exists for C++/ QML but I'm not sure about custom file formats).